content/batch/jobs.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Kubernetes Jobs"
+date: 2018-11-18T00:00:00-05:00
+weight: 20
+draft: false
+---
+
+### Kubernetes Jobs
+
+A _job_ creates one or more pods and ensures that a specified number of them successfully terminate. As pods successfully complete, the job tracks the successful completions. When a specified number of successful completions is reached, the job itself is complete. Deleting a Job will cleanup the pods it created.
+
+Run a sample Kubernetes Job using the `whalesay` image.
```

Comment:
Save the below manifest as 'job-whalesay.yaml' using your favorite editor

Reply:
Updating in next commit. Should we give an option to download the file directly?

Reply:
that would work too. its a nice addition not mandatory